src/app/tests/suites/certification/Test_TC_DM_2_2.yaml

```diff
@@ -50,14 +50,10 @@ tests:
       command: "readAttribute"
       attribute: "fabrics list"
       response:
-          value:
-              [
+          value: [
                   {
-                      FabricIndex: 0,
-                      RootPublicKey: 65,
-                      VendorID: 65521,
-                      FabricID: 0,
-                      NodeID: 305414945,
+                      # We really don't know what values to expect for any of
+                      # the fields, except Label.
```

Comment:
Maybe there is a need to use variables here ? (I still not have finished the patch for it thought, there are some variable scope issues with objective-c generated tests).

Reply:
We'd basically need to make sure the values here match whatever values ended up getting picked during commissioning.  I figured that was out of scope for this PR....